systems/plants/test/testManipulatorDynamics.m

```diff
@@ -58,44 +60,32 @@ function testActuatedPendulum()
 checkMex(m);
 end
 
-function regressionTestAtlasRPY()
-replaceMatFile = false;
-
-rng(23415, 'twister');
+function testAtlasRPY()
 
 r = createAtlas('rpy');
+r_newkinsol = setNewKinsolFlag(r,true);
 nq = r.getNumPositions();
 nv = r.getNumVelocities();
 
 nTests = 5;
-Hs = cell(nTests, 1);
-Cs = cell(nTests, 1);
-
 for i = 1 : nTests
   q = randn(nq, 1);
   v = randn(nv, 1);
   [H, C, B] = manipulatorDynamics(r, q, v, false);
-  Hs{i} = H;
-  Cs{i} = C;
-end
+  [H_new, C_new, B_new] = manipulatorDynamics(r_newkinsol, q, v, false);
 
-filename = 'regressionTestAtlasManipulatorDynamics.mat';
-if replaceMatFile
-  save(filename, varname(Hs), varname(Cs), varname(B));
-else
-  data = load(filename);
-  for i = 1 : nTests
-    valuecheck(data.Hs{i}, Hs{i}, 1e-10);
-    valuecheck(data.Cs{i}, Cs{i}, 1e-10);
-  end
-  valuecheck(data.B, B);
```

Comment:
You don't think that having a regression test adds anything anymore? It would tell us if something broke at any time, even after we've switched to the new implementation.